ocs_ci/ocs/ui/page_objects/page_navigator.py

```diff
@@ -338,14 +338,26 @@ def navigate_block_pool_page(self):
         Navigate to block pools page
 
         """
-        logger.info("Navigate to block pools page")
+        logger.info("Navigate to Block Pools Page")
         storage_system_details = (
             self.nav_odf_default_page()
             .nav_storage_systems_tab()
             .nav_storagecluster_storagesystem_details()
         )
         storage_system_details.nav_ceph_blockpool()
-        logger.info("Now at Block pool page")
+        logger.info("Now at Block Pools page")
+
+    def navigate_client_page(self):
+        """
```

Comment:
We need a new class for such functions. Page Navigator is the side-bar in the left side of the console. Can we create a class StorageCLients for the functionality available in the red square?

![image](https://github.com/red-hat-storage/ocs-ci/assets/61454420/d7888d8f-4ea9-426c-8d99-6e7b09fa1571)

Reply:
Yes, that's what I'm doing in client_ui.py file. I can rename it to storageclients if that's what you are suggesting.

Reply:
Yes please, def navigate_storage_clients_page should be good, wdyt?